### DISCUSSION TITLE
Why is the navigation bar only displaying four menus after initialization and startup in Skywalking version 9.7?

### DISCUSSION THREAD

**Discussion by 137-zwq** [apache/skywalking#11688] [Q&A] [answered]
![image](https://github.com/apache/skywalking/assets/64053611/dfd869db-a955-4272-b019-92cee86296f3)
Why is the navigation bar only displaying four menus after initialization and startup in Skywalking version 9.7?

**Comment by wu-sheng** [comment, marked as answer]
Only show menu if that layer has services available. 
All options are in the marketplace.

**Reply by 137-zwq** [reply]
I understand, thank you for your answer!
